common/src/main/java/com/github/twitch4j/common/util/EventManagerUtils.java

```diff
@@ -0,0 +1,52 @@
+package com.github.twitch4j.common.util;
+
+import com.github.philippheuer.events4j.core.EventManager;
+
+public class EventManagerUtils {
+
+    /**
+     * Validates the provided EventManager or initializes a new one
+     *
+     * @param eventManager EventManager
+     * @param defaultEventHandler The default eventHandler class
+     * @return EventManager
+     */
+    public static EventManager validateOrInitializeEventManager(EventManager eventManager, @SuppressWarnings("rawtypes") Class defaultEventHandler) {
+        if (eventManager == null) {
+            return initializeEventManager(defaultEventHandler);
```

Comment:
Should we also throw a RuntimeException if someone does `.withDefaultEventHandler(null)` and doesn't set a nonnull value for `eventManager`?

Reply:
Resolved this by also calling the validate method in case we initialize the EventManager in the library.